Algorithm/QCAlgorithm.cs

```diff
@@ -2257,7 +2257,7 @@ private T AddSecurity<T>(SecurityType securityType, string ticker, Resolution? r
                 symbol = QuantConnect.Symbol.Create(ticker, securityType, market);
             }
 
-            var configs = SubscriptionManager.SubscriptionDataConfigService.Add(symbol, resolution, fillDataForward, extendedMarketHours);
+            var configs = SubscriptionManager.SubscriptionDataConfigService.Add(symbol, resolution, fillDataForward, extendedMarketHours, dataNormalizationMode: UniverseSettings.DataNormalizationMode);
```

Comment:
Looks good 👍 I think potentially we could accept an argument to allow overriding this behavior if desired by the user, but we could go down that road if required/demanded